app/store/createStore.ts

```diff
@@ -72,4 +74,8 @@ const createStore = (
 export default createStore;
 
 export type Store = ReturnType<typeof createStore>;
-export type AppDispatch = Store['dispatch'];
+export type AppDispatch = ThunkDispatch<
+  RootState,
+  { getCookie: GetCookie },
+  AnyAction
+>;
```

Comment:
No. This is just bypassing the incorrect types we have elsewhere. This is not correct either.